.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Feature request
+about: Suggest an idea for this library
+
+---
+
+Thanks for stopping by to let us know something could be better!
+
+**PLEASE READ**: If you have a support contract with Google, please create an issue in the [support console](https://cloud.google.com/support/) instead of filing on GitHub. This will ensure a timely response.
+
+ **Is your feature request related to a problem? Please describe.**
```

Comment:
I don't think this formats the way you expect it to - the "view" button certainly doesn't show a nice version. I'm not quite sure what you're aiming for, but I suspect you want a lot more blank lines, maybe adding two spaces at the end of header lines to force a line break:

```
**Is your feature request related to a problem? Please describe.**  
A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

**Describe the solution you'd like**  
A clear and concise description of what you want to happen.

**Describe alternatives you've considered**  
A clear and concise description of any alternative solutions or features you've considered.

**Additional context**  
Add any other context or screenshots about the feature request here.
```

Is rendered as:

**Is your feature request related to a problem? Please describe.**  
A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

**Describe the solution you'd like**  
A clear and concise description of what you want to happen.

**Describe alternatives you've considered**  
A clear and concise description of any alternative solutions or features you've considered.

**Additional context**  
Add any other context or screenshots about the feature request here.

Note that there are two spaces at the end of each header line.

That's assuming the issue template Markdown works the same way as regular GitHub markdown, of course - maybe it doesn't?

Reply:
Thanks!  I added the new lines, but I'm not sure what the spaces at the end of the line would do.  GFM seems to handle the bold stuff without the extra spaces.  Also my editor deletes trailing whitespace lol.

Reply:
The extra spaces would force a line break. It's the difference between:

**Bold bit, no spaces**
Plain bit

and

**Bold bit, spaces at end**  
Plain bit

I think it might be worth just merging and seeing how it looks though. Let's get it right in this repo, then roll those changes out to the other repos.

Reply:
(Ignore me, this is just GFM doing line breaks differently to other markdown.)